app/classifier/tasks/slider/editor.jsx

```diff
@@ -0,0 +1,150 @@
+import { MarkdownEditor } from 'markdownz';
+import React from 'react';
+import alert from '../../../lib/alert';
+import AutoSave from '../../../components/auto-save';
+import handleInputChange from '../../../lib/handle-input-change';
+import MarkdownHelp from '../../../partials/markdown-help';
+import NextTaskSelector from '../next-task-selector';
+
+const SliderTaskEditor = (props) => {
+  const handleChange = handleInputChange.bind(props.workflow);
+  let nextTask;
+  let helpBox;
+  if (!props.isSubtask) {
+    nextTask = (
+      <AutoSave resource={props.workflow}>
+        <span className="form-label">Next task</span>
+        <br />
+        <NextTaskSelector
+          workflow={props.workflow}
+          name={`${props.taskPrefix}.next`}
+          value={props.task.next ? props.task.next : ''}
+          onChange={handleChange}
+        />
+      </AutoSave>
+    );
+    helpBox = (
+      <div>
+        <AutoSave resource={props.workflow}>
+          <span className="form-label">Help text</span>
+          <br />
+          <MarkdownEditor
+            name={`${props.taskPrefix}.help`}
+            value={props.task.help ? props.task.help : ''}
+            rows="4"
+            className="full"
+            onChange={handleChange}
+            onHelp={() => { return alert(<MarkdownHelp />); }}
+          />
+        </AutoSave>
+        <small className="form-help">Add text and images for a help window.</small>
+      </div>
+    );
+  }
+  return (
+    <div className="text-editor">
+      <section>
+        <div>
+          <AutoSave resource={props.workflow}>
+            <span className="form-label">Main text</span>
+            <br />
+            <textarea
+              name={`${props.taskPrefix}.instruction`}
+              value={props.task.instruction}
+              className="standard-input full"
+              onChange={handleChange}
+            />
+          </AutoSave>
+          <small className="form-help">Describe the task, or ask the question, in a way that is clear to a non-expert. You can use markdown to format this text.</small>
+          <br />
+          <br />
+          {helpBox}
+          <hr />
+          <span className="form-label">Slider Properties</span>
+          <br />
+          <small className="form-help">Specify the minimum, maximum, and default values for the slider. You can also specify the step size for the slider.</small>
+          <br />
+          <AutoSave resource={props.workflow}>
+            <label htmlFor="minInput">
+              <input
```

Comment:
This adds a box-shadow to the slider, which might look a little funky
<img width="488" alt="screen shot 2017-05-12 at 13 26 52" src="https://cloud.githubusercontent.com/assets/11770176/25998274/2bcff220-3718-11e7-9834-efe659d86e13.png">